src/nvim/ex_docmd.c

```diff
@@ -9239,6 +9239,18 @@ static int ses_do_win(win_T *wp)
   return true;
 }
 
+static int put_view_curpos(FILE *fd, const win_T *wp, char *spaces)
```

Comment:
I prefer the return type to be `bool` or `TriState`. `FAIL/OK/NOTDONE` share the same values as the enums in TriState.